dipy/viz/fvtk.py

```diff
@@ -79,6 +79,9 @@
     msg = "Python VTK is not installed"
     warn(msg)
 
+    ren, have_ren, _ = optional_package('dipy.viz.window.ren',
```

Comment:
@arokem I am not sure about that design decision. But there must be a reason for example it says `# Allow import, but disable doctests if we don't have vtk`, although I am not sure what that means. @Garyfallidis